src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class PostalCode {
+	private String value;
+	
```

Comment:
As discussed in class, you can also have regex validations for each of the values for each class.